### DISCUSSION TITLE
I need some assistance

### DISCUSSION THREAD

**Discussion by jmost2455** [kyleskom/NBA-Machine-Learning-Sports-Betting#361] [General]
Hi, I ran the main.py and got this;

2023-11-16 19:12:18.575505: I tensorflow/core/platform/cpu_feature_guard.cc:182] This TensorFlow binary is optimized to use available CPU instructions in performance-critical operations.
To enable the following instructions: SSE SSE2 SSE3 SSE4.1 SSE4.2 AVX AVX2 AVX512F AVX512_VNNI FMA, in other operations, rebuild TensorFlow with the appropriate compiler flags.
Miami Heat vs Brooklyn Nets: 

What do I enter next? I've tried o/u then odds but afterwards nothing happens

**Comment by nealmick** [comment]
The top error about the cpu docent matter its just warning, and says that normally.  The issue i think is its going into a no odds mode and is expecting you to input your own odds.  Make sure your running with this: python3 main.py -xgb -odds=fanduel

 it specifies the book you want to get odds from.

**Reply by jmost2455** [reply]
can you further explain "Make sure your running with this: python3 main.py -xgb -odds=fanduel"

**Reply by jmost2455** [reply]
nevermind I got it
